Confidence/src/main/java/com/spotify/confidence/DebugLogger.kt

```diff
@@ -35,6 +36,11 @@ internal class DebugLoggerImpl(private val filterLevel: LoggingLevel) : DebugLog
         verbose("[$action] $context")
     }
 
+    override fun logResolve(flag: String, context: Map<String, ConfidenceValue>) {
+        val trimContext = "$context".replace(" ", "")
```

Comment:
not sure you want to do this. 
first; think about what format is expected. probably some json?
also, if we trim context values from `" "` then any context key or value that contains a `" "` would be incorrect, right?

Reply:
Right, I will figure out how to translate the context into a JSON..